apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationList.jsx

```diff
@@ -13,8 +13,9 @@ const isReadFilterItems = [
 ]
 
 const reportsFilterItems = [
-  { label: django.gettext('Reported'), value: 'True' },
-  { label: django.gettext('All comments'), value: 'All' }
+  { label: django.gettext('Reported by AI'), value: 'ai' },
+  { label: django.gettext('Reported by Users'), value: 'users' },
+  { label: django.gettext('All comments'), value: 'off' }
```

Comment:
i think here needs a filter for both ai and user reports

Reply:
you mean the intersection?

Reply:
confirmed with max, all returns unfiltered all comments.